src/toffy/watcher_callbacks.py

```diff
@@ -179,6 +180,11 @@ def extract_tiffs(self, tiff_out_dir: str, panel: pd.DataFrame, **kwargs):
         if not os.path.exists(tiff_out_dir):
             os.makedirs(tiff_out_dir)
 
+        extracted_img_dir = os.path.join(tiff_out_dir, self.point_name)
+        if os.path.exists(extracted_img_dir):
+            warnings.warn(f"Images already extracted for FOV {self.point_name}")
+            return
+
```

Comment:
Is there any reason to believe that a FOV could be interrupted during the extraction step resulting in missing channel tiffs? If not, this is fine.

Reply:
It's hard to predict where in the extraction process for a FOV the run could be interrupted. I think it's safest to include a test for all cases just in case we don't leave behind any missing pieces.

Reply:
So then this needs to be updated then, right? This will skip extraction for all channels even if only one tiff was generated